src/ui/shared.js

```diff
@@ -443,19 +448,15 @@ window.importPreferences = async (file) => {
   const reader = new FileReader();
   reader.readAsText(file, 'UTF-8');
   reader.onload = async (e) => {
-    let preferences;
-    try {
-      preferences = JSON.parse(e.target.result);
-    } catch (error) {
-      // eslint-disable-next-line no-console
-      console.log('error while importing preferences', error);
-      showError('Error while importing preferences!');
-    }
```

Comment:
Why did you remove the catch / showError?

Reply:
I merged the two try/catch statements into one.

Reply:
Ah, makes sense. Thanks.